clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderFollowerStoreIngestionTask.java

```diff
@@ -473,6 +480,17 @@ protected void processConsumerAction(ConsumerAction message, Store store) throws
          */
         endSegment(partition);
         break;
+      case VERSION_ROLE_CHANGE_TO_TRIGGER_RESUBSCRIBE:
```

Comment:
BTW, shouldn't this be handled as a common action in SIT? I don't see any logic regarding leader follower role yet.